### ISSUE TITLE
tanzu CLI error output should say how to get to the usefulErrorMessage

### ISSUE THREAD

**Issue by ciberkleid** [vmware-tanzu/community-edition#3070]
## Feature Request

tanzu CLI outputs the following on errors:
`Delete failed: Error (see .status.usefulErrorMessage for details)`

The error would be more user-friendly if it included guidance on ho to get the status, for example, by running `tanzu package installed get <package-name>` (or if there is another advised way to get to the details, then that can be mentioned. This is the way I discovered by chance).

**Comment by stmcginnis** [comment]
Thanks Cora. Can you expand on this a little? Under what condition did you get to this error message?

**Comment by consolewitch** [comment]
@stmcginnis I don't think @ciberkleid is referencing a specific problem. She is pointing out the irony and futility of saying "see .status.usefulErrorMessage" without telling the user where or what .status.usefulErrorMessage is.

**Comment by stmcginnis** [comment]
Yep, just looking for how to get this to reproduce. Likely an issue for `tanzu-framework`, but would be good to understand how to trigger it.
